test/test_other.py

```diff
@@ -8779,37 +8783,41 @@ def test_side_module_folder_deps(self):
     self.run_process([EMCC, test_file('hello_world.c'), '-sMAIN_MODULE', '-o', 'main.js', '-L', 'subdir', '-lside2'])
 
   @is_slow_test
-  def test_lto(self):
+  @parameterized({
+    '01': (['-O1'],),
```

Comment:
Looks like we lost the no-arguments case? Is LTO without opts not worth testing I guess?